hack/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+source "$(dirname $0)/common"
+
+registry_ip=$(oc get service docker-registry -n default -o jsonpath={.spec.clusterIP})
+cat manifests/05-deployment.yaml | \
+    sed -e "s/quay\.io\/openshift\/cluster-logging-operator/${registry_ip}:5000\/openshift\/cluster-logging-operator/" | \
```

Comment:
Is there a template-like feature e.g. `oc process -p IMAGE_PREFIX=$registry_ip:5000/openshift/cluster-logging-operator`

You can specify a different sed delimiter like this, if you want to avoid the "leaning toothpick syndrome":
```
sed -e "s,quay\.io/openshift/cluster-logging-operator,${registry_ip}:5000/openshift/cluster-logging-operator," | \ 
```